docs/Installation.md

```diff
@@ -111,102 +113,20 @@ Check the [declaration of prerequisites] to make sure your preferred combination
+database] first. If you instead want to start from afresh, then go to [appendix
+C][Cleaning database] and remove the old database first.
 
 If you keep the database, skip the initialization of the Zonemaster database,
 but if you have removed the old Zonemaster database, then do the initialization.
 
 
-#### 3.2.1 Instructions for MariaDB (CentOS)
-
-Configure Zonemaster::Backend to use the correct database engine:
-
-```sh
-sudo sed -i '/\bengine\b/ s/=.*/= MySQL/' /etc/zonemaster/backend_config.ini
-```
-
-> **Note:** See the [backend configuration] documentation for details.
-
-Install, configure and start database engine:
-
-```sh
-sudo yum -y install mariadb-server
-sudo systemctl enable mariadb
-sudo systemctl start mariadb
-```
-
-Initialize the database (unless you keep an old database):
-
-```sh
-sudo mysql < $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/initial-mysql.sql
-```
-
-> **Note:** This creates a database called `zonemaster`, as well as a user
-> called "zonemaster" with the password "zonemaster" (as stated in the config
-> file). This user has just enough permissions to run the backend software.
-
-
-#### 3.2.2 Instructions for PostgreSQL (CentOS)
-
-Configure Zonemaster::Backend to use the correct database engine:
-
-```sh
-sudo sed -i '/\bengine\b/ s/=.*/= PostgreSQL/' /etc/zonemaster/backend_config.ini
-```
-
-> **Note:** See the [backend configuration] documentation for details.
-
-
-Install, configure and start database engine:
-
-* On CentOS 7:
-
-  ```sh
-  sudo rpm -iUvh https://yum.postgresql.org/9.3/redhat/rhel-7-x86_64/pgdg-centos93-9.3-3.noarch.rpm
-  sudo yum -y install postgresql93-server perl-DBD-Pg
-  sudo /usr/pgsql-9.3/bin/postgresql93-setup initdb
-  sudo sed -i '/^[^#]/ s/ident$/md5/' /var/lib/pgsql/9.3/data/pg_hba.conf
-  sudo systemctl enable postgresql-9.3
-  sudo systemctl start postgresql-9.3
-  ```
-
-* On CentOS 8:
-
-  ```sh
-  sudo yum -y install postgresql-server perl-DBD-Pg
-  sudo postgresql-setup --initdb --unit postgresql
-  sudo sed -i '/^[^#]/ s/ident$/md5/' /var/lib/pgsql/data/pg_hba.conf
-  sudo systemctl enable postgresql
-  sudo systemctl start postgresql
-  ```
-
-Initialize Zonemaster database (unless you keep an old database):
-
-```sh
-sudo -u postgres psql -f $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/initial-postgres.sql
-```
-
-> **Note:** This creates a database called `zonemaster`, as well as a user called
-> "zonemaster" with the password "zonemaster" (as stated in the config file).
-> This user has just enough permissions to run the backend software.
-
-
-#### 3.2.3 Instructions for SQLite (CentOS)
+#### 3.2.1 Instructions for SQLite (CentOS)
 
 > **Note:** Zonemaster with SQLite is not meant for an installation with heavy
 > load.
 
-Configure Zonemaster::Backend to use the correct database engine and database
-path:
-
-```sh
-sudo sed -i '/\bengine\b/ s/=.*/= SQLite/' /etc/zonemaster/backend_config.ini

```

Comment:
Yes, it makes it cleaner, but then there is no way back to SQLite if you have installed it with MySQL unless you copy the configuration file again. Could we move this command to the appendix part? Along with the equivalent commands for the other OSs.

Reply:
> unless you copy the configuration file again

Or edit it, which can be done using the "docs/Configuration.md".

> Could we move this command to the appendix part?

For me the purpose of the commands is to make it easy to install the Backend. Once installed, I think it is acceptable to let the user edit one config file to make some changes. So I would say that we gain very little to move this command to an appendix, especially since this is a just a tweak to avoid saying: «Open the backend_config.ini file and edit it to your needs, if you want to change the database engine, update the "DB.engine" parameter. For other parameters, see the docs/Configuration.md file». Having said that, I realize that such a sentence might be missing in the Installation document.

Reply:
I guess that is fine.